src/query/service/src/interpreters/interpreter_table_create.rs

```diff
@@ -347,6 +349,18 @@ impl CreateTableInterpreter {
             self.build_request(stat)
         }?;
 
+        let catalog_type = catalog.info().catalog_type();
+
+        if matches!(catalog_type, CatalogType::Default | CatalogType::Hive)
+            && (req.table_meta.iceberg_table_properties.is_some()
+                || req.table_meta.iceberg_partition.is_some())
```

Comment:
just use 

```
table_properties and iceberg_partition
```
These keeps the abstraction in table.

Reply:
How about 

iceberg_properties and iceberg_partition?

Reply:
I don't think it's good to have `iceberg_` prefix.

What if we will integrate with other catalog with partition suppport.

Reply:
Well, rigorous thinking, I'll change it later

Reply:
Done